### PR TITLE
Add Interlock Device Classes

### PR DESCRIPTION
**Pull request by xresende**

### CODE REVIEW

**Review by anacso17 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by fernandohds564 — APPROVED** [review]

File: siriuspy/siriuspy/devices/blctrl.py [hunk 1]
```diff
@@ -1,9 +1,46 @@
 """Beamline Control."""
```
Comment (anacso17) [inline siriuspy/siriuspy/devices/blctrl.py:1]: maybe we could rename this module to `ppsctrl`?

Comment (xresende) [reply siriuspy/siriuspy/devices/blctrl.py:1]: done

Comment (anacso17) [reply siriuspy/siriuspy/devices/blctrl.py]: @xresende , it seems the file was deleted

Comment (xresende) [reply siriuspy/siriuspy/devices/blctrl.py]: @it was renames to `intlkctrl.py` since now it has classes for machine MPS and PPS interlocks.

**Review by fernandohds564 — APPROVED** [review]

**Review by murilobalves — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Review by anacso17 — APPROVED** [review]

**Pull request merged**